cinema/serializers.py

```diff
@@ -14,11 +14,59 @@
 
     def update(self, instance, validated_data):
         instance.title = validated_data.get("title", instance.title)
-        instance.description = validated_data.get(
-            "description", instance.description
-        )
+        instance.description = validated_data.get("description",
+                                                  instance.description)
         instance.duration = validated_data.get("duration", instance.duration)
 
         instance.save()
 
         return instance
+
+
+class GenreSerializer(serializers.Serializer):
```

Comment:
Consider using `serializers.ModelSerializer` for `GenreSerializer` to simplify the code by automatically handling the `create` and `update` methods.